packages/common/src/dto/skill-information.dto.ts

```diff
@@ -26,11 +27,24 @@ export class SkillInformationDTO {
   registrationNumber!: number;
 
   @IsString()
-  @IsIn(Object.values(CurrentEmploymentTypes))
-  currentEmploymentType!: CurrentEmploymentTypes;
+  @IsIn(Object.values(EmploymentTypes))
+  currentEmployment!: EmploymentTypes;
+
+  specialties!: SpecialtyType[];
+
+  healthAuthorities!: HealthAuthorities[];
 
   @IsString()
-  @Length(1, 256)
+  @Length(1, 255)
   @IsOptional()
   additionalComments?: string;
 }
+
+export interface SpecialtyType {
```

Comment:
Should this be an additional DTO like `PlacementPreferencesDTO`?

Reply:
I think so. That would match the pattern already established.